checks.d/segfault.py

```diff
@@ -0,0 +1,101 @@
+# Add lib/ to the import path:
```

Comment:
Hrm, with this change you've effectively written an arbitrary regex-in-logfile detector, right? Should we combine these and just use multiple instances of the same check?

Reply:
They behave somewhat differently -- one emits "since last reboot" and one emits "count over a time window". There is significant overlap which I wouldn't be averse to trying to simplify, but I'd like to ship it so it can be utilized right away for now